### PR TITLE
CFP extension update

### PR DESCRIPTION
**Pull request by swapnilogale**
Also, a minor change to sponsor prospectus (changed date from 2020 to 2022)

### CODE REVIEW

**Comment by swapnilogale** [comment]
@ericholscher @plaindocs added a new announcement for CFP extension

File: docs/conf/australia/2021/news/cfp-extend.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+:template: {{year}}/generic.html
+
+.. post:: August 14, 2021
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cfp-extend.rst]:
```suggestion
.. post:: August 16, 2021
```

File: docs/conf/australia/2021/news/cfp-extend.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+===========================
+
+Greetings fellow documentarians!
+
+The conference is now just under four months away, and we hope you’re already planning for this year's remote installment of Write the Docs Australia and India conference.
+
+New deadline to submit your proposal is 31st Aug
+--------------------------------------------------
+
+The `Call for Proposals <https://www.writethedocs.org/conf/australia/2021/cfp/>`_ is now extended until **31st August 2021 at Midnight AEST** – We added two extra weeks
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cfp-extend.rst]: This para feels slightly repetitive? Dunno if you wanna tweak?

**Review by plaindocs — APPROVED** [review]
Ship it, or edit then ship! :-D

File: docs/conf/australia/2021/news/cfp-extend.rst (hunk 2 above)
Comment (swapnilogale) [reply docs/conf/australia/2021/news/cfp-extend.rst]: Done

**Comment by plaindocs** [comment]
:+1: :shipit:

**Pull request merged**